### PR TITLE
Remove experimental notice for vz, 9p, user-v2 net, grpc forwarder

### PR DESCRIPTION
**Pull request by AkihiroSuda**

### CODE REVIEW

File: examples/default.yaml [hunk 1]
```diff
@@ -126,8 +126,8 @@ mounts:
 # 🟢 Builtin default: null
 mountTypesUnsupported: null
 
-# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (EXPERIMENTAL until Lima v1.0, from QEMU’s virtio-9p-pci, aka virtfs),
-# or "virtiofs" (EXPERIMENTAL, needs `vmType: vz`; will graduate from experimental in Lima v1.0)
+# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (QEMU’s virtio-9p-pci, aka virtfs),
+# or "virtiofs".
```
Comment (jandubois) [inline examples/default.yaml:130]: Don't we want to keep the comment that `virtiofs` needs `vmType: vz`?

File: examples/default.yaml [hunk 2]
```diff
@@ -296,7 +296,7 @@ cpuType:
 #   x86_64: "qemu64" # (or "host,-pdpe1gb" when running on x86_64 host)
 
 rosetta:
-  # Enable Rosetta for Linux (EXPERIMENTAL; will graduate from experimental in Lima v1.0).
+  # Enable Rosetta for Linux
```
Comment (jandubois) [inline examples/default.yaml:299]:
```suggestion
  # Enable Rosetta inside the VM; needs `vmType: vz`
```

File: website/content/en/docs/config/mount/_index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ The default mount type is shown in the following table:
 | < 0.10           | reverse-sshfs + Builtin SFTP server                           |
 | >= 0.10          | reverse-sshfs + OpenSSH SFTP server                           |
 | >= 0.17          | reverse-sshfs + OpenSSH SFTP server for QEMU, virtiofs for VZ |
-| >= 1.0 (Planned) | 9p for QEMU, virtiofs for VZ                                  |
+| >= 1.0           | 9p for QEMU, virtiofs for VZ                                  |
```
Comment (jandubois) [inline website/content/en/docs/config/mount/_index.md:15]: You could shrink the width of the whole column now by 4 characters, if you want to.

**Review by jandubois — CHANGES_REQUESTED** [review]
LGTM, but I think we need to keep the hint that `virtiofs` requires `vz`.

File: examples/default.yaml (hunk 1 above)
Comment (AkihiroSuda) [reply examples/default.yaml:130]: No, on Linux

Comment (jandubois) [reply examples/default.yaml:130]:
```suggestion
# or "virtiofs" (needs `vmType: vz` on macOS).
```

Comment (AkihiroSuda) [reply examples/default.yaml]: Added "needs vz on macOS"

File: examples/default.yaml (hunk 2 above)
Comment (AkihiroSuda) [reply examples/default.yaml]: done

File: examples/default.yaml [hunk 4]
```diff
@@ -126,8 +126,8 @@ mounts:
 # 🟢 Builtin default: null
 mountTypesUnsupported: null
 
-# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (EXPERIMENTAL until Lima v1.0, from QEMU’s virtio-9p-pci, aka virtfs),
-# or "virtiofs" (EXPERIMENTAL, needs `vmType: vz`; will graduate from experimental in Lima v1.0)
+# Mount type for above mounts, such as "reverse-sshfs" (from sshocker), "9p" (QEMU’s virtio-9p-pci, aka virtfs),
+# or "virtiofs" (needs `vmType: vz` on macOS).
```
Comment (jandubois) [inline examples/default.yaml:130]: Just realized that `virtiofs` is still experimental on Linux; do you want to include that here?

```suggestion
# or "virtiofs" (experimental on Linux; needs `vmType: vz` on macOS).
```

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

File: examples/default.yaml (hunk 4 above)
Comment (AkihiroSuda) [reply examples/default.yaml]: done

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**